docs/usage/release-notes-for-major-versions.md

```diff
@@ -7,6 +7,49 @@ The most recent versions are always at the top of the page.
 This is because recent versions may revert changes made in an older version.
 You also don't have to scroll to the bottom of the page to find the latest release notes.
 
+## Version 38
+
+### Breaking changes for 38
+
+General:
+
+- Require Node.js 20 ([#30291](https://github.com/renovatebot/renovate/pull/30291))
+- The Renovate Docker images no longer have `-slim` tags. You must stop using the `-slim` prefix. Renovate now defaults to the `-slim` tag type behavior.
+
+Specific:
+
+- **bitbucket-server:** autodetect `gitAuthor`, if possible ([#29525](https://github.com/renovatebot/renovate/pull/29525))
+- **config:** change from `boolean` to `enum` for `onboardingNoDeps`. Renovate now onboards repositories with no dependencies, with one exception: if you run Renovate in `autodiscover` mode then you must manually onboard Renovate for repos with no dependencies
+- **config:** sanitize special characters from branch names for vulnerability type PRs. This may cause Renovate to autoclose/replace existing PRs
+- **config:** change the order of `globalExtends` resolution, it is applied _first_ and remaining global config takes precedence
+- **datasource/docker:** Docker Hub lookups prefers `hub.docker.com` over `index.docker.io`. To revert to the old behavior: set `RENOVATE_X_DOCKER_HUB_TAGS_DISABLE=true` in your env
+- **git:** check _all_ commits on the branch to decide if the branch was modified ([#28225](https://github.com/renovatebot/renovate/pull/28225))
+- **gitea:** use "bearer auth" instead of "token auth" to authenticate to the Gitea platform
+- **github:** if you run Renovate as a GitHub app then `platformCommit` is automatically enabled
```

Comment:
Do we want to mention that this "breaks" commit signing?

See:

- #30443

Reply:
Commits will be signed via the platform itself